cnf-tests/testsuites/pkg/pods/pods.go

```diff
@@ -322,8 +333,36 @@ func GetLog(p *corev1.Pod) (string, error) {
 	return buf.String(), nil
 }
 
-// ExecCommand runs command in the pod and returns buffer output
+// GetLog connects to a pod and fetches log from a specific container
+func GetLogForContainer(p *corev1.Pod, containerName string) (string, error) {
```

Comment:
can you refactor the GetLog to use this function on the first container in list?

Reply:
done!